blank/README.md

```diff
@@ -0,0 +1,21 @@
+# Blank Template (JavaScript)
+
+![npm (scoped)](https://img.shields.io/npm/v/@pxblue/react-native-template-blank?color=%23007bc1&label=%40pxblue%2Freact-native-template-blank)
+
+This is an official PX Blue template used internally by the [PX Blue CLI](https://www.npmjs.com/package/@pxblue/cli).
+
+This template installs the basic PX Blue libraries for themes and components and configures the application to use them. It includes a simple placeholder screen that you can replace with your own application content.
+
+## Usage
+This template can be used with the PX Blue CLI:
+```sh
+npx -p @pxblue/cli pxb new react-native --name=myapp --cli=rnc --language=js --template=blank
+```
+
+## Project Structure
+Projects created using this template will start out with the following file structure:
+
+```
+└── root
+└── App.js                             // a sample landing page
```

Comment:
I would not put root here since there is no folder called root in the generated project. I would go with something like:

```
|── /ios                                   // ios project folder
|── /android                               // android project folder
|── /assets                                // fonts and images used by the application
└── App.jsx                                // sample landing page
```